src/storage/src/store.rs

```diff
@@ -183,10 +184,21 @@ pub trait StateStoreIter: Send + 'static {
 pub struct ReadOptions {
     pub epoch: u64,
     pub table_id: Option<TableId>,
+    pub ttl: Option<u32>,
 }
 
 #[derive(Default, Clone)]
 pub struct WriteOptions {
     pub epoch: u64,
     pub table_id: TableId,
 }
+
+impl ReadOptions {
+    pub fn min_epoch(&self) -> u64 {
+        let epoch = Epoch(self.epoch);
+        match self.ttl {
+            Some(ttl_u32) => epoch.min_epoch_with_interval(ttl_u32 as u64).0,
```

Comment:
ttl_u32 is measure in second? we should convert it to ms to do the epoch subtraction.

Reply:
unify the unit of ttl to second